tests/schemas/status.schema.js

```diff
@@ -15,15 +15,6 @@
  */
 import Joi from 'joi';
 
-const statusSchema = {
```

Comment:
Please revert this change as we still need this.

Reply:
applied.